### ISSUE TITLE
--config doesn't work in GUNICORN_CMD_ARGS

### ISSUE THREAD

**Issue by Code0x58** [benoitc/gunicorn#1576]
Specifying `--config` in the `GUNICORN_CMD_ARGS` environment variable quietly fails as config file loading only happens when it is passed as an argument:
https://github.com/benoitc/gunicorn/blob/328e509260ae70de6c04c5ba885ee17960b3ced5/gunicorn/app/base.py#L137-L175

**Comment by tilgovi** [comment]
It looks like we parse the command options from the environment only after loading from a config file. We would have to parse these first, to get the config file option.

Good find.

**Comment by benoitc** [comment]
yeah the env should probably checked first and then the rest. At least it's seems to be the common pattern. Would you mind to provide a PR for it?

**Comment by Code0x58** [comment]
Pull request #1581 submitted.
